testsuite/MDAnalysisTests/coordinates/test_netcdf.py

```diff
@@ -299,6 +300,23 @@ def test_box(self, universe, index, expected):
         assert_almost_equal(self.box_refs[expected], universe.dimensions)
 
 
+class TestNCDFReader4(object):
+    """NCDF Trajectory exported by cpptaj, without `time` variable.
+    """
+    prec = 3
+
+    @pytest.fixture(scope='class')
+    def u(self):
+        return mda.Universe(CPPTRAJ_TRAJ_TOP,
+                            [CPPTRAJ_TRAJ, CPPTRAJ_TRAJ])
+
+    def test_positions_1(self, u):
```

Comment:
```suggestion
    def test_chain_times(self, u):
```
positions_i is a little bit confusing for a name given you are checking times

Reply:
You are right, it was a leftover from a wild copy/paste :-)